src/test/java/com/coursesPlatform/coursePortfolio/CourseTest.java

```diff
@@ -40,10 +41,44 @@ public void shouldReturnAllCourses() {
         CategoryDTO category = new CategoryDTO("", "");
         CourseDTO course = new CourseDTO("", "");
         categoryService.add(category);
-        courseService.add(course,category.getTitle());
+        courseService.add(course, category.getTitle());
         //when
         List<CourseDTO> allCourses = courseService.findAllCourses();
         //then
         assertThat(allCourses).contains(course);
     }
+
+    @Test
+    void shouldReturnLecturesFromCourse() {
+        //given
```

Comment:
This test is nice, and still can be improved - how about preparing private methods responsible for creating AND saving specific object? That way given clause will shorten itself